app/src/main/java/com/simplecity/amp_library/ui/presenters/PlayerPresenter.java

```diff
@@ -124,11 +129,15 @@ public void bindView(@NonNull PlayerView view) {
     }
 
 
-    private void refreshCurrentTimeText(long playbackTime) {
+    private void refreshTimeText(long playbackTime) {
         if (playbackTime != currentPlaybackTime) {
             PlayerView view = getView();
             if (view != null) {
                 view.currentTimeChanged(playbackTime);
+                if(SettingsManager.getInstance().preferRemainingTime())
+                    view.totalTimeChanged(totalTime/1000 - playbackTime);
```

Comment:
Remaining time should have a `-` in front of it. I would suggest achieving that via:
`view.totalTimeChanged(-(totalTime/1000 - playbackTime))`